examples/mnist/how_to_build.md

```diff
@@ -0,0 +1,25 @@
+<p align="center"><b>How to build MNIST example program.</b><p align="center">
+
+
+
+<b>Step:1</b> Install Cmake from https://cmake.org/install/  
+<b>Step:2</b> Open Terminal.  
+<b>Step:3</b>
+```bash
+cd install_path # adjust `install_path` to where you want to install tiny-dnn
+git clone https://github.com/tiny-dnn/tiny-dnn.git  
+cd tiny-dnn  
+mkdir build & cd build   
+CMake -DBUILD_EXAMPLES=ON .. & make  
+cd examples   
+./example_mnist_train ../../data  
+```
+Now training will start.  
+After training is finished, in terminal type  
+```
+./example_mnist_test sample_img.bmp   
```

Comment:
Beginners may not understand how to get `sample_img.bmp`.
So introducing the file url written in `readme.md` might be helpful.